### ISSUE TITLE
Get v5.0.2 code through cocoaPods is incorrect

### ISSUE THREAD

**Issue by peterhoucll** [dropbox/dropbox-sdk-obj-c#276]
I got the SDK through cocoaPods, and the code I got is as follows:
### DBClientsManager.h
```
+ (void)handleRedirectURL:(NSURL *)url completion:(DBOAuthCompletion)completion;
+ (void)handleRedirectURLTeam:(NSURL *)url completion:(DBOAuthCompletion)completion;
```
But your example is called like this:
```
BOOL canHandle = [DBClientsManager handleRedirectURL:url completion:completion];
return canHandle;
```

### podfile:
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '10.3'

target 'xxx' do
    pod 'ObjectiveDropboxOfficial'
end

### Console:
$ pod install
Installing ObjectiveDropboxOfficial (5.0.2)
Generating Pods project
Integrating client project
Pod installation complete! There are XX dependencies from the Podfile and XX total pods installed.

**Comment by Dids** [comment]
Looks like the `README.md` is just outdated, but the API docs are up to date.

```objc
#import <ObjectiveDropboxOfficial/ObjectiveDropboxOfficial.h>

- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url
            options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
  DBOAuthCompletion completion = ^(DBOAuthResult *authResult) {
    if (authResult != nil) {
      if ([authResult isSuccess]) {
        NSLog(@"\n\nSuccess! User is logged into Dropbox.\n\n");
      } else if ([authResult isCancel]) {
        NSLog(@"\n\nAuthorization flow was manually canceled by user!\n\n");
      } else if ([authResult isError]) {
        NSLog(@"\n\nError: %@\n\n", authResult);
      }
    }
  };
  [DBClientsManager handleRedirectURL:url completion:completion];
  return NO;
}
```

^ From here: https://dropbox.github.io/dropbox-sdk-obj-c/api-docs/latest/

**Comment by peterhoucll** [comment]
thank you!

**Comment by greg-db** [comment]
Thanks for the report and help! I'll ask the team to fix this up. (The README actually shows the usage as of the latest version of the code on master, but that hasn't included in a release yet.)

**Comment by peterhoucll** [comment]
you mean that README.md is the latest, but the code of V5.0.2 is not the latest?

**Comment by Dids** [comment]
[README.md on master](https://github.com/dropbox/dropbox-sdk-obj-c/blob/master/README.md) is still using the old syntax: https://github.com/dropbox/dropbox-sdk-obj-c#ios-5

**Comment by greg-db** [comment]
To clarify, the "new" functionality is that `handleRedirectURL` now returns a `BOOL` that indicates whether the URL can be handled. This functionality was added [here](https://github.com/dropbox/dropbox-sdk-obj-c/commit/42d88e238cb0dbb730c735e31fa55d95a0df8dab), after the v5.0.2 release, and so attempting to use that in <=v5.0.2 failed. The README was updated along with that change, so it did not match the functionality included with v5.0.2 release and docs.

We've now released [v5.0.3](https://github.com/dropbox/dropbox-sdk-obj-c/releases/tag/5.0.3) (though the docs are still on lagging on 5.0.2; we'll get those updated shortly) which does include this functionality, so you can now use the new syntax, where `handleRedirectURL` now returns a `BOOL`, once you update to v5.0.3. (You can technically still use the old syntax too, even once you update to v5.0.3, since the return value would just be ignored.)
